packages/paste-website/stories/TokenCard.stories.tsx

```diff
@@ -193,6 +193,12 @@ SizingTokenSquare.args = {
   name: "size-square-200",
 };
 
+export const SizingTokenBase = Template.bind({});
+SizingTokenBase.args = {
+  category: "sizings",
+  name: "size-base-90",
+};
```

Comment:
question: Just for my understanding, is this being used anywhere? Curious why it's been added in if it's not used in a story.

Reply:
It is being used in the storybook, eg: https://paste-storybook.twilio.design/?path=/story/website-token-card--sizing-token

Reply:
praise: nice catch adding this story!

when we revamped our tokens list page, we created all these stories to make sure the token card designs don't regress with specific token categories